playground/src/api/category/content-types/category/schema.json

```diff
@@ -23,7 +26,12 @@
       "inversedBy": "category"
     },
     "title": {
-      "type": "string"
+      "type": "string",
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      }
```

Comment:
It’s better if you don’t put localization on dor the categories.
That way we can test both localized and non-localized content types.

I’m pretty sure that’s why the unit tests are failing now.